project-security/verifying-signatures.md

```diff
@@ -92,38 +92,56 @@ There are several ways to get the Qubes Master Signing Key.
-Here are some ideas:
-
- - Use the PGP Web of Trust.
- - Check the key against different keyservers.
- - Use different search engines to search for the fingerprint.
- - Use Tor to view and search for the fingerprint on various websites.
- - Use various VPNs and proxy servers.
- - Use different Wi-Fi networks (work, school, internet cafe, etc.).
- - Ask people to post the fingerprint in various forums and chat rooms.
- - Check against PDFs and photographs in which the fingerprint appears
-   (e.g., slides from a talk or on a T-shirt).
- - Repeat all of the above from different computers and devices.
+Once you have obtained the Qubes Master Signing Key, you must verify that it is authentic rather than a forgery.
+Anyone can create a PGP key with the name "Qubes Master Signing Key," so you cannot rely on the name alone.
+You also should not rely on any single website, not even over HTTPS.
 
+So, what *should* you do?
+One option is to use the PGP [Web of Trust](https://en.wikipedia.org/wiki/Web_of_trust).
 In addition, some operating systems have built-in keyrings containing keys capable of validating the Qubes Master Signing Key.
 For example, if you have a Debian system, then your keyring may already contain the necessary keys.
 
-For additional security, we also publish the fingerprint of the Qubes Master Signing Key here (but [remember not to blindly trust the live version of this website][website-trust]):
+Another option is to rely on the key's fingerprint.
+Every PGP key has a fingerprint that uniquely identifies it among all PGP keys (viewable with `gpg2 --fingerprint <KEY_ID>`).
+Therefore, if you know the genuine Qubes Master Signing Key fingerprint, then you always have an easy way to confirm whether any purported copy of it is authentic, simply by comparing the fingerprints.
+
+For example, here is the Qubes Master Signing Key fingerprint:
 
     pub   4096R/36879494 2010-04-01
           Key fingerprint = 427F 11FD 0FAA 4B08 0123  F01C DDFA 1A3E 3687 9494
     uid   Qubes Master Signing Key
 
-Once you're confident that you have the legitimate Qubes Master Signing Key, set its trust level to "ultimate" so that it can be used to automatically verify all the keys signed by the Qubes Master Signing Key (in particular, Release Signing Keys).
+But how do you know that this is the real fingerprint?
+After all, [this website could be compromised][website-trust], so the fingerprint you see here may not be genuine.
+That's why we strongly suggest obtaining the fingerprint from *multiple, independent sources in several different ways*.
+
+Here are some ideas for how to do that:
+
+ - Download the key from different keyservers.
+ - Use different search engines to [search](https://duckduckgo.com/?q=%22427F+11FD+0FAA+4B08+0123+F01C+DDFA+1A3E+3687+9494%22) for the fingerprint.
```

Comment:
Those two are not really useful. If you search for a fake key based on its fingerprint, you will get that very fake key. Searching with key name could be a better idea, but it's far to easy to promote anything in a search engine, so I'd simply remove those two.

Reply:
Removed